src/interfaces/manifest-transport.ts

```diff
@@ -1,12 +1,12 @@
 import {VariablesAssignments} from "./package-manager.interfaces";
 
 export interface ManifestNodeTransport {
+    packageKeyAndVersion: string;
     packageKey: string;
     packageId: string,
-    packageVersion: string;
     variables: ManifestVariable[],
     space: ManifestSpace,
-    dependencies: ManifestDependency[]
+    dependenciesByVersion: Map<string, ManifestDependency[]>;
```

Comment:
is there any possibility to pass dependency without specifying the version? Or should it be pointed out even if the latest version will be used?